lambda/app/src/routes.ts

```diff
@@ -42,22 +42,14 @@ const responseHeaders = {
 
 const BASE_PATH = '/app';
 
-const redirect = (res, url, status = 301) => {
-  const headers = { Location: url };
-  if (isFunction(res.headers)) res.headers(headers);
-  else res.set(headers);
-  return res.status(status).send();
+const handleError = (res, err) => {
+  console.error(err);
+  res.status(422).json({ success: false, message: err.message || err });
 };
 
 export const setRoutes = (app: any) => {
   app.use((req, res, next) => {
-    try {
```

Comment:
What was this doing originally?  Am I misunderstanding that it was always just setting the responseHeaders?

Reply:
main logic is the same in line `52` it sets the default headers still.
`    res.set(responseHeaders);`